shell/components/form/KeyValue.vue

```diff
@@ -635,6 +658,16 @@ export default {
             <div v-else-if="row.binary">
               {{ binaryTextSize(row.value) }}
             </div>
+            <CodeMirror
+              v-else-if="valueMarkdownMultiline"
+              ref="cm"
+              :class="{['focus']: cmFocus[row[keyName]]}"
+              :value="row[valueName]"
+              :options="cmOptions"
+              :mode="mode"
+              @onInput="onInputMarkdownMultiline(row[keyName], $event)"
+              @onFocus="onFocusMarkdownMultiline(row[keyName], $event)"
```

Comment:
Can't we simply use the focusing functionalities of the DOM without add something extra?
It will get also an issue with A11y as well in the future.

Reply:
I think is not possible because the focus of the `value` fields depends on CodeMirror component which a nested child.

Reply:
I've tried with the DOM focus attribute and is on fact not possible to easily focus on the component.
We'll still have a bunch of A11y issues though, not a problem for now.